src/execution/subscribe.ts

```diff
@@ -163,14 +120,50 @@ function mapSourceToResponse(
  * "Supporting Subscriptions at Scale" information in the GraphQL specification.
  */
 export function createSourceEventStream(
-  schema: GraphQLSchema,
-  document: DocumentNode,
+  argsOrSchema: ExecutionArgs | GraphQLSchema,
+  /** Note: document argument is required if positional arguments are used */
+  /** @deprecated will be removed in next major version in favor of named arguments */
```

Comment:
Please make separate TS signatures as we did for GraphQLErrors.
That way single `deprecate` would be enough, also it will be less confusing.

Reply:
Great idea. [e8a2a74](https://github.com/graphql/graphql-js/pull/3634/commits/e8a2a7405c921b549124db8f7a53c241f8b375b2)